pdftools/_cli.py

```diff
@@ -0,0 +1,273 @@
+    parser_add.add_argument("src", type=str, default=None, help="PDF source file")
+    parser_add.add_argument(
+        "-p",
+        "--pages",
+        nargs="+",
+        help="list of pages to add to the output. Examples: 5; 1-9; 1-; -9",
+    )
+    # output
+    parser_add.add_argument(
+        "-o",
+        "--output",
+        type=str,
+        default=None,
+        help="Name of the output file. If None, the `dest` file will be overwritten",
+    )
+
+    # Copy
+    # --------------------------------------------
+    parser_copy = SUBPARSERS.add_parser(
+        "copy",
+        help="Copy specific pages of a PDF file in a new file",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+    parser_copy.add_argument(
+        "src", type=str, default=None, help="Source PDF containing pages to copy"
+    )
+    parser_copy.add_argument(
+        "-o",
+        "--output",
+        type=str,
+        default=None,
+        help="Name of the output file. If None, the `dest` file will be overwritten",
+    )
+    parser_copy.add_argument(
+        "-p",
+        "--pages",
+        dest="pages",
+        type=str,
+        nargs="+",
+        default=1,
+        help="list of pages to copy in the new file. "
+        "Examples: \n"
+        '"5 8 10": Pages 5, 8, 10; '
+        '"1-9":    Pages 1 to 9; '
+        '"5-":     Pages from 5 to last page; '
+        '"-9":     Pages from beginning to 9',
+    )
+    parser_copy.add_argument("-y", action="store_true", help="yes to all")
+
+    # Insert
+    # --------------------------------------------
+    parser_insert = SUBPARSERS.add_parser(
+        "insert",
+        help="Insert pages of one file into another",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+    parser_insert.add_argument("dest", type=str, help="Destination PDF file")
+    parser_insert.add_argument("src", type=str, help="Source PDF file")
+    parser_insert.add_argument(
+        "-o",
+        "--output",
+        type=str,
+        default=None,
+        help="Name of the output file. If None, the `dest` file will be overwritten",
+    )
+    parser_insert.add_argument(
+        "-p",
+        "--pages",
+        nargs="+",
+        help="List of page numbers (start with 1) which will be inserted. If None, all pages will be inserted (default). Examples: 5; 1-9; 1-; -9",
+    )
+    parser_insert.add_argument(
+        "-i",
+        "--index",
+        type=int,
+        default=None,
+        help="Page number (1-indexed) of destination file where the pages will be inserted. If None they will be added at the end of the file",
+    )
+
+    # Merge
+    # --------------------------------------------
+    parser_merge = SUBPARSERS.add_parser(
+        "merge",
+        help="Merge the pages of multiple input files into one output file",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+    parser_merge.add_argument(
```

Comment:
For all other commands the output file is defined by the -o/--output argument. This should be the same here, too.

Reply:
Makes sense. I think I left it as a required argument because you need an output file name to write to, but adding a default, as I've done in the most recent commit, should take care of this